server/server.js

```diff
@@ -12,8 +15,43 @@ var corsOptions = {
   optionsSuccessStatus: 200, // some legacy browsers (IE11, various SmartTVs) choke on 204
 };
 
+const api = bizSdk.FacebookAdsApi.init(process.env.ACCESS_TOKEN);
```

Comment:
Let's move this logic into routes.js, also let's remove console.logs when you push the code